playbook/workshops/dodge/blank_canvas.md

```diff
@@ -0,0 +1,204 @@
+
+```js
+function setup() {
+  createCanvas(300, 500);
+}
+
+function draw() {
+  background(255, 240, 220);
+}
+```
+
+In this part, we won't talk about what `function setup()` and `function draw()`
+are. We'll this in more detail [later](linear_player_movement.md#understanding-the-code).
+
+## Tinkering with the values of `createCanvas`
+
+![](img/checkmark.png) In your **_working bin_**, try changing some of the values of `createCanvas` to
+try to see what it changes.
+
+Here is an example of how I play around with the values of `createCanvas` to try  understand what it
+does:
+
+> ![](img/t1_change_canvas_values.gif)
+
+## Understanding `createCanvas`
+
+As a reminder, the area on which everything lives is called the
+"canvas":
+
+> ![](img/r_vocab2.png)
+
+`createCanvas` creates and sets the size of the canvas so:
+
+```
+createCanvas(300, 500);
+              ^    ^
+              |    └ set the *height* of the canvas to 400
+              |
+              └ set the *width* of canvas  to 600
+```
+
+Here is a diagram representing the result of the above code:
+
+![](img/t1_canvas_dimensions.png)
+
+<!-- Google Drawing https://docs.google.com/drawings/d/1c0rai1S-RTuSrfU_WTnUKDH8o3c-eaeobzf7O58ad0k/edit?usp=sharing -->
+
+## Tinkering with the values of `background`
+
+![](img/checkmark.png) Try changing some of the values of `createCanvas` to
+try to see what it changes.
+
+Here is an example of how I play around with the values of `background` to try
+understand what it does:
+
+> ![](img/t1_background_color_change.gif)
+
+## Understanding `createCanvas`
+
+```
+This code sets the background color of the canvas:
+
+  background(255, 255, 255);
+              ^    ^    ^
+              |    |    └ set how much *red* there is
+              |    |    
+              |    └ set how much *green* there is
+              |
+              └ set how much *blue* there is
+```
+
+## Setting the Proper Dimensions
+
+Now that you've gotten a chance to experiment, we want our canvas to have a
+width of 300 and a height of 500.
+
+![](img/checkmark.png) Go ahead and make sure that your setup function looks
+like this:
+
+```js
+function setup() {
+  createCanvas(300, 500);
+}
+```
+
+## Recap
+
+- You learned how to use `createCanvas`
+- You learned how to use `background`
+
+## Next Up
+
+| **[![](img/sq_2_add_player_sprite.png)    <br> 2: Adding the Player Sprite]    (add_player_sprite.md)**    |
+|:----------------------------------------------------------------------------------------------------|
+
+---
+
+# Appendix
+
+## Documentation ![](img/documentation.png)
+
+**Links to the official p5.js Documentation**
+
+- [`background(255, 255, 255)`](http://p5js.org/reference/#/p5/background)
+- [`createCanvas(300, 500);`](http://p5js.org/reference/#/p5/createCanvas)
+
+## Computer Science Fundamentals ![](img/computer_science.png)
+
+**_This Section is Coming Soon!_**—In the mean time, here is a sneak peak:
```

Comment:
Please use regular dashes `-` and not em/endashes :-) Em/endashes are not very monospace friendly.